lib/class-wp-rest-menu-items-controller.php

```diff
@@ -181,7 +181,7 @@ public function create_item( $request ) {
 		}
 		$prepared_nav_item = (array) $prepared_nav_item;
 
-		$nav_menu_item_id = wp_update_nav_menu_item( $prepared_nav_item['menu-id'], $prepared_nav_item['menu-item-db-id'], $prepared_nav_item );
+		$nav_menu_item_id = wp_update_nav_menu_item( wp_slash( $prepared_nav_item['menu-id'] ), wp_slash( $prepared_nav_item['menu-item-db-id'] ), wp_slash( $prepared_nav_item ) );
```

Comment:
Why do we need to slash menu id and db id here? Feels like overkill.

Reply:
Good catch! Fixed in 3c36175618efccbcafb9dab6d4d9bfc54c741fb5